src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -240,6 +250,35 @@ func (s *commitLogSource) Read(
 				return bootstrap.NamespaceResults{}, err
 			}
 		}
+
+		// Read index snapshot files
+		indexSnapshotFiles, err := s.indexSnapshotFilesFn(
```

Comment:
are we going to be reading snapshot files twice now (like the commit log)?

Reply:
Yea, unfortunately :/. I think I can piggy back off of nate's work to cache read info file results between bootstrap runs to reduce this work after it lands: https://github.com/m3db/m3/pull/2598/files

Reply:
I think this is also true for calls to `ReadIndexInfofiles`, ya? Either way, probably worth doing in a follow up PR so as to not expand the size of this one.